shared/build.gradle.kts

```diff
@@ -101,6 +97,12 @@ kotlin {
     sourceSets["iosMain"].dependencies {
         implementation(Deps.SqlDelight.driverIos)
         implementation(Deps.Ktor.ios)
+
+        implementation(Deps.Coroutines.common) {
+            version {
+                strictly(Versions.coroutines)
```

Comment:
Do we even need this anymore? I think ktor was the one causing issues

Reply:
apparently, yes. Without it we get tests hanging due missing run loop, which indicates it's not picking up the native-mt version

Reply:
@russhwolf @kpgalligan  hello there.. Firstly, thank you both so much for all your hard work.  KMM is amazing and I'm telling everyone that will listen that it's the future of mobile :-)  

Ok, now the bad news :-(  I have a fork of this repo that I've been using to convert an app I have in development. After I pulled down this merged PR, my iOS build started getting runtime errors. 

It seems to be when I'm creating a cold flow for a KTOR network call.  I tried rolling back just coroutines/kotlin version bumps but that wasn't enough.  I can provide more information as I dig more, but I wanted to just let you know what I discovered.. Thank you again!


2021-07-04 11:01:17.729169-0400 tRacket[39241:897272] Verbose: (Network) RESPONSE https://io.adafruit.com/api/v2/ndipatri/feeds/occupancy/data?start_time=2021-05-135T11:01-0400&x-aio-key=aio_XmSy92vYjddoZhQ4f19ztFc8Qm6r failed with exception: kotlin.native.concurrent.InvalidMutabilityException: mutation attempt of frozen kotlin.collections.HashMap@393d508
2021-07-04 11:01:17.730368-0400 tRacket[39241:897272] Error: (OccupancyModel) Error downloading data
2021-07-04 11:01:17.743070-0400 tRacket[39241:897272] kotlin.native.concurrent.InvalidMutabilityException: mutation attempt of frozen kotlin.collections.HashMap@393d508
        at 0   tRacket                             0x000000010d7f721f kfun:kotlin.Throwable#<init>(kotlin.String?){} + 95
        at 1   tRacket                             0x000000010d7ef97d kfun:kotlin.Exception#<init>(kotlin.String?){} + 93
        at 2   tRacket                             0x000000010d7efbed kfun:kotlin.RuntimeException#<init>(kotlin.String?){} + 93
        at 3   tRacket                             0x000000010d8278fd kfun:kotlin.native.concurrent.InvalidMutabilityException#<init>(kotlin.String){} + 93
        at 4   tRacket                             0x000000010d8290ff ThrowInvalidMutabilityException + 431
        at 5   tRacket                             0x000000010d994590 MutationCheck + 128
        at 6   tRacket                             0x000000010d80a102 kfun:kotlin.collections.HashMap.<set-length>#internal + 82